framework/include/base/MooseFunctor.h

```diff
@@ -586,7 +586,7 @@ class FunctorBase
   mutable std::map<SingleSidedFaceArg, ValueType> _ssf_arg_to_value;
 
   /// name of the functor
-  const MooseFunctorName _functor_name;
+  MooseFunctorName _functor_name;
```

Comment:
Looks like could keep this `const` now which I really appreciate

Reply:
ok I ll add that back in a future PR if you dont mind.

Reply:
yes that's fine

Reply:
or I'll do it in the advection schemes PR